README.md

```diff
@@ -49,7 +55,9 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+
+guest << residents[1]
```

Comment:
Did you try this?  I expect you'll get an error.  This code attempts to push "Belle" to a new (uninitialized) variable.  Please review accessing and assigning to ruby hashes.